compiler/resolution/visibleFunctions.cpp

```diff
@@ -869,7 +837,7 @@ static void getVisibleFunctionsImpl(const char*       name,
 
   // Need to continue going up in case our parent scopes also had private
   // uses that were skipped.
-  if (block != rootModule->block) {
+  {
```

Comment:
Ditto here about the rootModule block stuff

Reply:
Right, same situation.